DragaliaAPI/Program.cs

```diff
@@ -24,6 +27,15 @@
 
 WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
 
+Stopwatch watch = new();
+Console.WriteLine("Loading MasterAsset data.");
+
+watch.Start();
+RuntimeHelpers.RunClassConstructor(typeof(MasterAsset).TypeHandle);
+watch.Stop();
+
+Console.WriteLine($"Loaded MasterAsset in {watch}.");
```

Comment:
Could move this down to after we build the app and use app.logger -- then it will go to seq and you can have structured logging. 

Not that important for a minor performance diagnostic I suppose

Reply:
addressed